linode/manager
# M3-5990: Remove nvme from volumes landing page

## Description 📝

After Fremont was upgraded from `HDD` to `NVMe`, storage across all regions is now `NVMe` 
This PR removes the UI `NVMe` chip component from the `Volumes` landing page.


## Preview 📷

![image00](https://user-images.githubusercontent.com/119514965/207630736-c5279ce7-7e6b-48e1-a8f4-4bda69e6d10f.jpg)


## How to test 🧪
- Go to the Volumes landing page.
- Visually inspect the UI to verify the UI chip component has been removed.


## Please note ⚠️
-  The `hardware_type` property is now deprecated from `api-v4` after Fremont was upgraded from `HDD` to `NVMe`.  Storage across all regions will be `NVMe`.
